### DISCUSSION TITLE
染疫不報，研究：美國半數人曾撒謊

### DISCUSSION THREAD

**Discussion by ABCDEFGHIJKLMNO12345678** [1101-GS4719-C/Python-Community-GS4719-C#769] [General]
隨者疫情日益嚴重 越來越多人都感染了新冠肺炎 然而一但感染了就必需在家格離 從剛開始的14天到現在的7天 與其在家待上一到兩週 許多人乾脆就不通報了 但也要呼籲大家 有病就通報 在家好好休息 別影響到他人

https://technews.tw/2022/10/12/people-dishonest-about-covid-related-status/

**Comment by williamyen0623** [comment]
疫情黑樹，BlackTree

**Comment by manipulateed** [comment]
這也是台灣現在的現象，選擇沈默而不去影響自己的生活。

**Comment by Pietra1226** [comment]
因為代價太高，我想通報後受到輿論抨擊是讓大家選擇不通報的主因。

**Comment by evanchou1204** [comment]
有一說一，我也曾經是防疫黑數，沒辦法，通報要停課七天，真的太麻煩。

**Comment by danwan1027** [comment]
台灣目前似乎也是在面臨一樣的問題...

**Comment by bonny123865** [comment]
感覺現在確診都讓大家在道德跟現實作選擇!
因為隔離或是密切接觸都要關在家不能出外工作或是買東西，所以感覺很多人不願意承認都是希望自己不要被拘束?!

**Comment by Doreen331** [comment]
我身邊就一堆黑數，雖然不通報某程度來說有點不道德，但通報了確實很麻煩，加上大家對於確診已經習以為常，所以很多人都會選擇不通報，甚至連診所的醫生都只依照一般感冒來治療

**Comment by Kotori0501** [comment]
感謝分享
我身邊的親戚就有人沒通報，一方面是通報有走的流程太麻煩，或是我身旁的親戚就有人是從東南亞國家來的，也沒學過中文，因此有些國字看不懂，另一方面通報後就必須強制關3-7天，如果是自己開店的話，這幾天就等於沒錢賺了。

**Comment by dabbie109303501** [comment]
學校有些課程根本沒有線上的管道，即使有，也跟在現場的教學品質差很多，而且要是剛好碰到期中考，考試權益很可能會受損，所以才會有很多黑數(而且只要不篩誰知道你是感冒還是染疫)

**Comment by Chiway1229** [comment]
很抱歉，我之前就當過防疫黑數
畢竟各種政策跟規定實在是太影響生活了

**Comment by yuxinlin93** [comment]
我也通說過很多人染疫卻沒通報，原因是感覺報了會更麻煩，還會影響到很多東西，像是工作、生活，所以最後選擇自己在家慢慢康復就好了

**Comment by ru0372572** [comment]
我覺得隔離真的很麻煩，事實上應該沒必要這麼久，還要搭8000多塊的計程車回家隔(雖然不用自己出)

**Comment by Zoe06060606** [comment]
通報真的太虧了 還要在家裡關7天 還要接受別人的眼光

**Comment by CPCpyt** [comment]
的確通報真的對人沒好處，
尤其是家裡有事業的情況，
通報讓整間公司都暫停營業，
那個損失真的不划算。

**Comment by StevenWanglolz** [comment]
我的室友之前也是防疫黑數，因為通報的話會影響更多人，所以我們只能默默承受

**Comment by Evan0979** [comment]
畢竟染一已經快成為生活中無法避免的一部份,通報對生活的影響實在太大,唉

**Comment by peggy0329** [comment]
比起很多人不通報，我更好奇為什麼研究顯示年輕男性比女性更不容易誠實通報？
心理學上有什麼研究結果與這個結果相關嗎？

**Comment by dinosaur1009** [comment]
確診黑數隨著疫情蔓延的情況越來越多了 在沒有危及生命安全的情況下 很多人選擇不通報....

**Comment by ghost** [comment]
對於感染者，選擇沈默而不去影響自己的生活；而對於非感染者，這些都是他們將承擔的風險

**Comment by swerfg03joe** [comment]
正常，因誘因不夠，沒有人會想影響自己生活而去通報，這市政府需要去衡量的，該怎麼獎勵確診補貼隔離時造成的經濟損失，或是直接頒布嚴格處罰條例並鼓勵檢舉。嚴格處罰條例並鼓勵檢舉。
